src/server/config.js

```diff
@@ -51,17 +36,14 @@ function loadBabelConfig(babelConfigPath) {
 export default function (configType, baseConfig, configDir) {
   const config = baseConfig;
 
-  // Search for a .babelrc in the config directory, then the module root
+  // Search for a .babelrc starting in the config directory
   // directory. If found, use that to extend webpack configurations.
-  let babelConfig = loadBabelConfig(path.resolve(configDir, '.babelrc'));
-  let inConfigDir = true;
-
-  if (!babelConfig) {
-    babelConfig = loadBabelConfig('.babelrc');
```

Comment:
@tleunen it could be. I don't know the reason some people just do it.
May be you need to do another lookup of `find-babel-config` with depth=0 for the config directory.